src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureFlagOptions.cs

```diff
@@ -28,6 +28,12 @@ public class FeatureFlagOptions
         /// </summary>
         public TimeSpan CacheExpirationInterval { get; set; } = RefreshConstants.DefaultFeatureFlagsCacheExpirationInterval;
 
+        /// <summary>
+        /// Forces feature flags and dynamic features to be written into the configuration system using the  
+        /// feature management V2.0 schema introduced in Microsoft.FeatureManagement 3.0.0-preview library.
+        /// </summary>
+        public bool EnableFeatureManagementV2SchemaPreview { get; set; } = false;
```

Comment:
This API should only be in preview versions of the library. I'm wondering if there is a way to break the build if a stable version is produced with this API exposed. Perhaps a test case could do it?

Reply:
Not applicable anymore.